scripts/templates/train-template.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#SBATCH --account vjgo8416-mod-sim-2  # Only required if you are a member of more than one Baskerville project
```

Comment:
pass in this val to generation script for flexibility

Reply:
done